swarmrl/networks/flax_network.py

```diff
@@ -164,8 +164,11 @@ def compute_action(self, observables: List, explore_mode: bool = False):
         # Add a small value to the log_probs to avoid log(0) errors.
         eps = 1e-8
         log_probs = np.log(jax.nn.softmax(logits) + eps)
-        if explore_mode:
-            indices = self.exploration_policy(indices, len(logits))
+
+        # if explore_mode:
```

Comment:
was this on purpose?

Reply:
Deliberately commented out but should also be deleted.

Reply:
Resolved now.